web/src/inspector.js

```diff
@@ -170,11 +178,54 @@ export class Inspector extends EventTarget {
     };
   }
 
+  static _createSettingsPage() {
```

Comment:
Is there a reason the elements are written in JS and not in HTML?

Reply:
I am the wrong person to ask. I reused the existing design of what was already there.

Building the page and keeping direct references to important elements probably has performance benefits over constantly looking them up by ID, but I'm not sure how significant they are. If there are gotchas that led to this specific design, I don't know what they are.